pkg/log/fake_logger.go

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
That is fine; sorry for not being explicit: my question was if it's necessary to have a separate file for the fake logger? Is this better than having it inside the concrete tests file?